content/en/dashboards/widgets/wildcard.md

```diff
@@ -123,13 +123,13 @@ Click on the **Define Visual** tab to view how this query maps to Vega-Lite. Ope
 
 | Select data configuration  | Define Visual Specification |
 | ---  | ----------- |
-|{{< img src="/dashboards/widgets/wildcard/example_configuration_no_alias.png" alt="Example widget configuration, showing open data preview" style="width:100%;" >}} | {{< img src="/dashboards/widgets/wildcard/example_vega_spec_map_to_config.png" alt="Vega specification mapping the widget configuration field query1 to the vega field" style="width:100%;" >}}|
+|{{< img src="/dashboards/widgets/wildcard/example_configuration_no_alias.png" alt="Example widget configuration, showing open data preview" style="width:100%;" >}} | {{< img src="/dashboards/widgets/wildcard/define_visual_run_button.png" alt="Vega specification mapping the widget configuration field query1 to the vega field" style="width:100%;" >}}|
```

Comment:
New pictures look good! Should we delete the unused pictures? I figure we are renaming them  rather than replacing them so that we ensure people load the latest picture and don't have caching issues.

https://github.com/DataDog/documentation/blob/master/static/images/dashboards/widgets/wildcard/example_vega_spec_map_to_config.png

https://github.com/DataDog/documentation/blob/master/static/images/dashboards/widgets/wildcard/example_vega_spec_mismatch.png

As a stylistic note, I think the previous file names were more descriptive of the file content (mentioning mismatch and map-to-config), but ultimately the alt text is the most important part for the user.

Reply:
No the we run a script to delete unused images!